src/query/api/v1/handler/options.go

```diff
@@ -60,7 +60,13 @@ type ServiceOptions struct {
 	M3Agg *M3AggServiceOptions
 
 	DryRun bool
-	Force bool
+	Force  bool
+}
+
+// ServiceOptionsDefaults is the defaults for optional service options.
+type ServiceOptionsDefaults struct {
```

Comment:
Heh this is probably my bad - I think this isn't used anymore, perhaps remove this struct?